### PR TITLE
*: refine task.yaml sample file

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

refine the sample file of _task.yaml_.

### What is changed and how it works?

- add comments to explain the schema/table name matching _upstream_ or _downstream_
- add comments to explain the matching rule is _wildcard characters_ or _regular expression_
- make some config items update to date

### Check List 

Tests 

 - No code

Related changes

 - Need to update the documentation

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GregoryIan @WangXiangUSTC PTAL

File: dm/dm-ansible/conf/task.yaml.example [hunk 1]
```diff
@@ -69,7 +69,7 @@ mysql-instances:             # one or more source database, config more source d
 
     loader:                  # local loader rule
       pool-size: 16
-      dir: "./dumped_data"   # must be unique between tasks for the same instance
+      dir: "./dumped_data"   # if not ends with `.name`, then `.name` will be appended
```
Comment (amyangfei) [inline dm/dm-ansible/conf/task.yaml.example]: explain `name` is the name variable in task.yaml ?

Comment (csuzhangxc) [reply dm/dm-ansible/conf/task.yaml.example]: explained in https://github.com/pingcap/dm/pull/183/commits/b7aba30a615bb13a078766f9a8343b3730d31d15.

**Pull request merged**